django_project/arduino_air_quality_backend/arduino_air_quality_backend/settings.py

```diff
@@ -87,16 +88,20 @@
 
 AUTH_PASSWORD_VALIDATORS = [
     {
-        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+        'NAME': 'django.contrib.auth.password_validation'
+                '.UserAttributeSimilarityValidator',
```

Comment:
Je li ovo ovako razdvojeno radi kada pokrenes `python manage.py runserver`? ja sam to radio sa plusom

Reply:
To sam napravio da skratim liniju koda na <79 charactera, tj. Pycharm mi je ponudio reformat na takav način isprobao sam i radilo je pa sam ostavio tako...